### PR TITLE
Security Policy.

### PR DESCRIPTION
**Pull request by jmlait**
New security policy

### CODE REVIEW

File: tsc/process/security.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+The last two options may be surprising.  VDBs, however, are designed
+as open-ended containers of production data that may be terabytes in size.
+
+It is a bug if some file causes the library to crash.  It is a serious
+security issue if some file causes arbitrary code execution.
+
+***Runtime Library Expectations***
+
+We consider the library to run with the same privilage as the linked
```
Comment (danrbailey) [inline tsc/process/security.md]: typo - privilage

File: tsc/process/security.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+code.  As such, we do not guarantee any safety against malformed
+arguments.   Provided functions are called with well-formed
+parameters, we expect the same set of behaviours as with file
```
Comment (danrbailey) [inline tsc/process/security.md]: I struggle a lot with this rule - but aren't we supposed to be using American English? Ie behavior vs behaviour.

File: tsc/process/security.md (hunk 1 above)
Comment (jmlait) [reply tsc/process/security.md]: I had it spelled even worse in the original :>  When I fixed I obviously didn't fix far enough :>

File: tsc/process/security.md (hunk 2 above)
Comment (jmlait) [reply tsc/process/security.md]: Canadian English allows the use of either at the author's discretion.  Like you, my default is to keep the 'u's.  Which leads to embarrassments like the neighbour() function in VEX.  Americanized it in any case.

File: tsc/process/security.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+The best practice for building a clean VDB is populate an
+empty grid voxel-by-voxel with the desired data and only
+copy known and trusted metadata fields.
+
+****Inactive Voxels****
+
+When voxels are marked inactive in the grid, they are not cleared
+to the background value.  If you rely on the data being deleted, you
+should set it as well as setting it inactive.  In addition, calling
```
Comment (pcucka) [inline tsc/process/security.md]: How about "you should overwrite voxels' values as well as deactivating them"?

File: tsc/process/security.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
+**Security and OpenVDB**
+
+The OpenVDB project takes security very seriously.
```
Comment (pcucka) [inline tsc/process/security.md]: "The OpenVDB Technical Steering Committee (TSC) takes security very seriously" might be a better way to phrase this.

**Comment by danrbailey** [comment]
@jmlait - I think you missed a dco.

**Comment by danrbailey** [comment]
@kmuseth - think we're just waiting on your approval here, would you have a few minutes to review? There's a couple of CII best practice items we can tick off once this gets merged. :)

File: tsc/process/security.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+****Steganographic****
+
+Most image-based steganographic techniques can be applied to VDBs.
+Narrow band SDF have an additional concern, however.   Since only
+the zero crossing affects the perceived geometry, there is considerable
+room for information hiding in the off-band voxels.  Conversion to
+polygon and reconstruction from the polygonal mesh should eliminate
+those channels.
+
```
Comment (kmuseth) [inline tsc/process/security.md]: I would emphasize that by design most of the algorithms provided in the OpenVDB library ignore inactive values. Narrow band level sets is just one example of this type of data. As such virtually any VDB grid could be constructed to contain hidden information encoded as inactive values.

Comment (kmuseth) [reply tsc/process/security.md]: The easiest way to mediate this is to first run tools::changeBackground, which sets all inactive values to the same value, followed by tools::pruneInactive, which eliminates nodes with inactive values only (note that this method will ignore any nodes with active values, which is why the first method must be called as well)

**Pull request merged**